core/src/main/java/com/klaytn/caver/contract/Contract.java

```diff
@@ -0,0 +1,392 @@
+package com.klaytn.caver.contract;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.klaytn.caver.Caver;
+import com.klaytn.caver.abi.ABI;
+import com.klaytn.caver.methods.request.KlayLogFilter;
+import com.klaytn.caver.methods.response.Bytes32;
+import com.klaytn.caver.methods.response.KlayLogs;
+import com.klaytn.caver.methods.response.TransactionReceipt;
+import com.klaytn.caver.transaction.response.PollingTransactionReceiptProcessor;
+import com.klaytn.caver.transaction.response.TransactionReceiptProcessor;
+import com.klaytn.caver.transaction.type.SmartContractDeploy;
+import com.klaytn.caver.utils.CodeFormat;
+import com.klaytn.caver.utils.Utils;
+import io.reactivex.Flowable;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.functions.Consumer;
+import org.web3j.abi.datatypes.Type;
+import org.web3j.protocol.ObjectMapperFactory;
+import org.web3j.protocol.core.Request;
+import org.web3j.protocol.core.methods.response.EthSubscribe;
+import org.web3j.protocol.exceptions.TransactionException;
+import org.web3j.protocol.websocket.events.LogNotification;
+
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Contract {
+
+    /**
+     * A caver instance.
+     */
+    Caver caver;
+
+    /**
+     * A contract ABI(Application Binary interface) json string.
+     */
+    String abi;
+
+    /**
+     * A contract address.
+     */
+    String contractAddress;
+
+    /**
+     * The map where method name string and ContractMethod mapped.
+     */
+    Map<String, ContractMethod> methods;
+
+    /**
+     * The map where event name string and ContractEvent mapped.
+     */
+    Map<String, ContractEvent> events;
+
+    /**
+     * The ContractMethod instance related Contract's constructor.
+     */
+    ContractMethod constructor;
+
+    /**
+     * The default send option. When you execute call() or send() without SendOptions, defaultSendOptions will be used.
+     */
+    SendOptions defaultSendOptions;
```

Comment:
Do you think changing this name to `options` will be ambiguous?

Reply:
I think "options" is ambiguous.